Source/OxyPlot.GtkSharp.Shared/PlotView.cs

```diff
@@ -285,20 +285,28 @@ public void ShowTracker(TrackerHitResult data)
                 // us to set the background color
                 Gtk.EventBox labelHolder = new Gtk.EventBox();
                 this.trackerLabel = new Gtk.Label();
-                this.trackerLabel.SetPadding(3, 3);
+                this.trackerLabel.Xpad = 3;
+                this.trackerLabel.Ypad = 3;
                 OxyColor bgColor = OxyColors.LightSkyBlue;
-                labelHolder.ModifyBg(StateType.Normal, new Gdk.Color(bgColor.R, bgColor.G, bgColor.B));
+                //labelHolder.ModifyBg(StateType.Normal, new Gdk.Color(bgColor.R, bgColor.G, bgColor.B));
                 labelHolder.Add(this.trackerLabel);
                 this.Add(labelHolder);
                 labelHolder.ShowAll();
             }
             this.trackerLabel.Parent.Visible = true;
             this.trackerLabel.Text = data.ToString();
+#if GTKSHARP3
+            int pw = trackerLabel.Parent.AllocatedWidth;
+            int ph = trackerLabel.Parent.AllocatedHeight;
+            int xPos = (int)data.Position.X - pw / 2;
+            int yPos = (int)data.Position.Y - ph;
+            xPos = Math.Max(0, Math.Min(xPos, this.Allocation.Width - pw));
+            yPos = Math.Max(0, Math.Min(yPos, this.Allocation.Height - ph));
+#else
```

Comment:
What's the idea here?

Reply:
@hol430 does this look like something we might want to include on-top of your work?

I'm tempted to get on and merge #16, and pick up the pieces in separate PR (e.g. retarget to 2.1 proper; retarget example library). I'm still not in a good position to test it, but I'm reassured seeing that your work here overlaps with #10.

Reply:
@hol430 thanks so much for taking the time to look at this.